simbad/rotsearch/amore_search.py

```diff
@@ -427,8 +433,17 @@ def rot_succeeded_log(self, log):
                            process_all=True,
                            submit_qtype=self.submit_qtype,
                            submit_queue=self.submit_queue)
+            mr_log = os.path.join(output_dir, pdb, "mr", self.mr_program, pdb + "_mr.log")
             refmac_log = os.path.join(output_dir, pdb, "mr", self.mr_program, "refine", pdb + "_ref.log")
             if os.path.isfile(refmac_log):
                 refmac_parser = simbad.parsers.refmac_parser.RefmacParser(refmac_log)
-                return simbad.rotsearch.mr_job_succeeded(refmac_parser.final_r_fact, refmac_parser.final_r_free)
+                if simbad.mr._refinement_succeeded(refmac_parser.final_r_fact, refmac_parser.final_r_free):
```

Comment:
Do you want to try/except the parsing so it doesn't crash the whole program?

Reply:
The parser returns default values of 1 for R-fact/R-free so even if the refmac log is passed incorrectly the program doesn't crash.